mcs/build/tests.make

```diff
@@ -157,6 +157,9 @@ $(test_makefrag): $(test_response)
 
 -include $(test_makefrag)
 
+build-test-lib: $(test_lib)
```

Comment:
That would be great if I wanted to run the tests. I don't, I want to build the test lib.